conda_smithy/_version.py

```diff
@@ -1,4 +1,3 @@
-
 # This file helps to compute a version number in source trees obtained from
```

Comment:
Can you remove black from running on this file? This is a vendored file I believe.

Reply:
It's installed by versioneer. That said, why not update its style as well?